index.js

```diff
@@ -28,13 +28,24 @@ var crypto = require('crypto');
 
 
 // Defaults
+var server = "share1.dexcom.com"
+var bridge = readENV('BRIDGE_SERVER')
+    if (bridge && bridge.indexOf(".") > 1) {
+    server = bridge;
+   } 
+    else if (bridge && bridge === 'EU') {
+        server = "shareous1.dexcom.com";
+    } else {
+        server = "share1.dexcom.com";
```

Comment:
unnecessary assignment

Reply:
Could you elaborate?

Also, any reason why we couldn’t relax the strict comparison on “EU”? Fairly common problem is folks entering this value in different case.

(I think the case sensitivity should probably be relaxed *everywhere*, but might as well start here while we’re in it...)

Reply:
The value is already assigned above, there’s no need to re-assign the same value again

Reply:
fixed